functions/Restore-DbaDatabase.ps1

```diff
@@ -519,7 +519,7 @@ function Restore-DbaDatabase {
             if ($DestinationDataDirectory -ne '') {
                 if ((Test-DbaSqlPath -Path $DestinationDataDirectory -SqlInstance $SqlInstance -SqlCredential $SqlCredential) -ne $true) {
                     if ((New-DbaSqlDirectory -Path $DestinationDataDirectory -SqlInstance $SqlInstance -SqlCredential $SqlCredential).Created -ne $true) {
-                        Write-Warning "$FunctionName - DestinationDataDirectory $DestinationDataDirectory does not exist, and could not be created on $SqlInstance"
+                        Write-Message -Level Warning -Message "$FunctionName - DestinationDataDirectory $DestinationDataDirectory does not exist, and could not be created on $SqlInstance" 
                         break
```

Comment:
Use Stop-Function instead. Including the function name in messages is no longer needed.